apidef/oas/schema/x-tyk-gateway.md

```diff
@@ -529,6 +529,42 @@ Path is the path to shared object file in case of gopluign mode or path to js co
 **Field: `rawBodyOnly` (`boolean`)**
 RawBodyOnly if set to true, do not fill body in request or response object.
 
+**Field: `idExtractor` ([IDExtractor](#idextractor))**
+IDExtractor  configures ID extractor with coprocess custom authentication.
```

Comment:
I'm looking at the double space here and I don't know, is it our generator? doesn't gofmt clean it from comments?

Reply:
It's because of the comments, updating it.